client/elements/menus/more-par-menu.html

```diff
@@ -76,20 +86,43 @@
                   },
                   loadingParallels: {
                       type: Boolean,
-                      value: true
+                  },
+                  gotParallels: {
```

Comment:
I'd name it `parallelsAvailable` or `areParallelsAvailable`. We never used the past tense in variable names.